lib/mrkdwn.ml

```diff
@@ -1,14 +1,14 @@
 open Omd
-open Base
 
 (* https://api.slack.com/reference/surfaces/formatting#escaping *)
 let escape_mrkdwn =
-  String.concat_map ~f:(function
+  let c_map = function
     | '<' -> "&lt;"
     | '>' -> "&gt;"
     | '&' -> "&amp;"
     | c -> String.make 1 c
-    )
+  in
+  ExtLib.String.fold_left (fun s c -> s ^ c_map c) ""
```

Comment:
concat_map can be implemented efficiently, while fold_left with `^` inside cannot. There is `Stre.catmap` in devkit